drivers/lsm303agr/src/lib.rs

```diff
@@ -0,0 +1,123 @@
+#![no_std]
```

Comment:
I think this'll need a license header.

Reply:
or maybe not, the rust crates specify their license in Cargo.toml.

Reply:
Yes; that's what also the relevant tools tap into (eg. `cargo).